### PR TITLE
feat: added Image and input components

### PR DESCRIPTION
**Pull request by sharath-io**
deploy preview: https://deploy-preview-5--atom-ui.netlify.app/components/image/image.html

Screen Shot 2022-02-17 at 6 18 33 PM
Screen Shot 2022-02-17 at 6 53 08 PM

### CODE REVIEW

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+.img-responsive {
+    width: 100%; 
+    height: auto;
+    object-fit: cover;
+    object-position: center;
+}
+
+.img-round {
+    border-radius: 50%;
+}
```
Comment (LordUttam) [inline components/image/image.css:15]: add equal height and width or look into aspect-ratio property to make image actually round.

**Review by nyyshaaa — APPROVED** [review]
you can make the image round by giving it equal width and height for the round part, all else looks good to me.

**Review by LordUttam — COMMENTED** [review]
Try to make the same change in .round class in styles.css. Rest looks good to me.

File: components/image/image.css (hunk 1 above)
Comment (sharath-io) [reply components/image/image.css:15]: Updated the class. Thank you

**Pull request merged**